src/parser/github-comment-module.ts

```diff
@@ -0,0 +1,217 @@
+        return content.join("");
+      }
+
+      function generateContributionRow(
+        view: string,
+        contribution: string,
+        count: number,
+        reward: number | Decimal | undefined
+      ) {
+        return `
+          <tr>
+            <td>${view}</td>
+            <td>${contribution}</td>
+            <td>${count}</td>
+            <td>${reward || "-"}</td>
+          </tr>`;
+      }
+
+      if (result.task?.reward) {
+        content.push(generateContributionRow("Issue", "Task", 1, result.task.reward));
+      }
+      if (sorted.issues.task) {
+        content.push(generateContributionRow("Issue", "Specification", 1, sorted.issues.task.score?.reward));
+      }
+      if (sorted.issues.comments.length) {
+        content.push(
+          generateContributionRow(
+            "Issue",
+            "Comment",
+            sorted.issues.comments.length,
+            sorted.issues.comments.reduce((acc, curr) => acc.add(curr.score?.reward ?? 0), new Decimal(0))
+          )
+        );
+      }
+      if (sorted.reviews.length) {
+        content.push(
+          generateContributionRow(
+            "Review",
+            "Comment",
+            sorted.reviews.length,
+            sorted.reviews.reduce((acc, curr) => acc.add(curr.score?.reward ?? 0), new Decimal(0))
+          )
+        );
+      }
+      return content.join("");
+    }
+
+    function createIncentiveRows() {
+      const content: string[] = [];
+
+      if (!sorted) {
+        return content.join("");
+      }
+
+      function buildIncentiveRow(commentScore: GithubCommentScore) {
+        // Properly escape carriage returns for HTML rendering
+        const formatting = stringify(commentScore.score?.formatting?.content).replace(/[\n\r]/g, "&#13;");
+        return `
+          <tr>
+            <td>
+              <h6>
+                <a href="${commentScore.url}" target="_blank" rel="noopener">${commentScore.content.replace(/(.{64})..+/, "$1…")}</a>
+              </h6>
+            </td>
+            <td>
+            <details>
+              <summary>
+                ${Object.values(commentScore.score?.formatting?.content || {}).reduce((acc, curr) => {
+                  return acc.add(curr.score * curr.count);
+                }, new Decimal(0))}
+              </summary>
+              <pre>${formatting}</pre>
+             </details>
+            </td>
+            <td>${commentScore.score?.relevance || "-"}</td>
+            <td>${commentScore.score?.reward || "-"}</td>
+          </tr>`;
+      }
+
+      for (const issueComment of sorted.issues.comments) {
+        content.push(buildIncentiveRow(issueComment));
+      }
+      for (const reviewComment of sorted.reviews) {
+        content.push(buildIncentiveRow(reviewComment));
+      }
+      return content.join("");
+    }
+
+    return `
+    <details>
+      <summary>
+        <b>
+          <h3>
+            <a href="${result.permitUrl}" target="_blank" rel="noopener">
```

Comment:
I assume this is a permit for task assignee, but where are other permits or is that not done yet?

Reply:
There is one permit generated per participating user, last run:
https://github.com/Meniole/test/issues/1#issuecomment-2069443938

Reply:
gotcha, I thought this html is for all users